sympy/parsing/latex/_parse_latex_antlr.py

```diff
@@ -119,7 +119,8 @@ def convert_add(add):
     elif add.SUB():
         lh = convert_add(add.additive(0))
         rh = convert_add(add.additive(1))
-        return sympy.Add(lh, -1 * rh, evaluate=False)
```

Comment:
I suggest that you use `sympy.core.mul._keep_coeff` so you don't get an unecessary unevaluated Mul for when `rh` is a Rational or a product with a leading Rational like `2*x`. Do either of these cases apply here?

Reply:
Maybe with `sign=True`.

Reply:
I was wondering...but I think you would want `1 - 2x` to appear like that instead of as `1 + (-1)*2*x` ... not sure what will happen.

Reply:
The printer works around it but it is parsed as `1 + (-1)*2*x`; a `convert_mul` analogous to `convert_add` has to be added.